core/monster.py

```diff
@@ -398,7 +405,16 @@ def forward(self, image1, image2, iters=12, flow_init=None, test_mode=False):
         net_list = [torch.tanh(x[0]) for x in cnet_list]
         inp_list = [torch.relu(x[1]) for x in cnet_list]
         inp_list = [torch.relu(x) for x in inp_list]
-        inp_list = [list(conv(i).split(split_size=conv.out_channels//3, dim=1)) for i,conv in zip(inp_list, self.context_zqr_convs)]
+        # inp_list = [list(conv(i).split(split_size=conv.out_channels//3, dim=1)) for i,conv in zip(inp_list, self.context_zqr_convs)]
+
+        # Length = 3
+        result: List[List[torch.Tensor]] = []
+        for idx, conv in enumerate(self.context_zqr_convs):
+            x = inp_list[idx]
+            split_tensor = conv(x).split(split_size=conv.out_channels // 3, dim=1)
+            result.append([s for s in split_tensor])
```

Comment:
I guess you can just call `tolist()`, instead of doing the list comprehension.
```suggestion
            result.append([s for s in split_tensor])
```

Reply:
`result.append(list(split_tensor))` works